spdx/v2/v2_3/tagvalue/reader/parse_document.go

```diff
@@ -0,0 +1,22 @@
+package reader
+
+import (
+	"fmt"
+	"github.com/spdx/tools-golang/spdx/v2/v2_3"
+)
+
+func (parser *tvParser) parsePairForDocument(tag string, value string) error {
+	if parser.doc == nil {
+		return fmt.Errorf("no document struct created in parser ann pointer")
+	}
+
+	switch tag {
+	case "DocumentComment":
+		parser.doc = &v2_3.Document{}
```

Comment:
I'm not sure this is the right thing to do here -- if `parser.doc` is already set, all values will be overwritten with defaults

Reply:
Fixed, thanks!